src/itn_uptime_analyzer/identity.go

```diff
@@ -19,8 +19,8 @@ import (
 // The graphQLPort parameter should be optional but Golang doesn't permit that
 // As a workaround we use a pointer that can be defined as nil
 type Identity struct {
-	id, publicKey, publicIp, uptime string
-	graphQLPort                     *string
+	id, publicKey, publicIp string
+	graphQLPort, uptime     *string
```

Comment:
Is `uptime` a pointer now because it is an optional field? Also, why is `graphQLPort` a string and not an `int`?

Reply:
I don't have a valid answer for this. I am a bad developer.